doc/source/manual/state_snapshot_transfer.rst

```diff
@@ -84,8 +66,6 @@ Using mysqldump
 This method uses the standard :program:`mysqldump` utility
```

Comment:
This entire section for mysqldump should be removed also.  For 8.0, the mysqldump method has been removed.

Reply:
Ready for review